source/core/parameters.cc

```diff
@@ -225,9 +238,11 @@ namespace Parameters
         {
           std::runtime_error("Invalid output control scheme");
         }
-      dt             = prm.get_double("time step");
-      timeEnd        = prm.get_double("time end");
-      adapt          = prm.get_bool("adapt");
+      dt      = prm.get_double("time step");
+      timeEnd = prm.get_double("time end");
```

Comment:
time to refactor this parameter. 
```suggestion
      time_end = prm.get_double("time end");
```